tools/driver/p4c_src/p4c.bmv2.cfg

```diff
@@ -20,10 +20,19 @@ class BMV2Backend(BackendDriver):
     def __init__(self, target, arch, argParser):
         BackendDriver.__init__(self, target, arch, argParser)
 
+        prog = 'p4c-bm2-ss'
+        if arch == 'psa':
+            prog = 'p4c-bm2-psa'
+        elif arch == 'pna':
+            prog = 'p4c-bm2-pna'
+        elif arch != 'v1model':
```

Comment:
Nit: Would write this slightly differently. Since variables can break out of their scope in Python I would have v1model just be another elif branch and then let things fail with the `else`. A little easier to read.

Reply:
Agreed. I've made this change.